supplier_app/migrations/0001_initial.py

```diff
@@ -77,6 +81,8 @@ class Migration(migrations.Migration):
                 ('cuit', models.CharField(max_length=200)),
                 ('justificacion', models.CharField(max_length=200)),
                 ('forma_de_pago', models.CharField(max_length=200)),
+                ('inscripcion_AFIP_file', models.FileField(upload_to='file', validators=[django.core.validators.FileExtensionValidator(allowed_extensions=['pdf'])], verbose_name='Constancia de inscripción de AFIP')),
```

Comment:
Please review imports handling

Reply:
I don't manage that because when a type "./manage.py migrate" then django create this file